Mathlib/Geometry/Manifold/VectorBundle/Tangent.lean

```diff
@@ -411,7 +411,7 @@ theorem tangentBundleModelSpaceHomeomorph_coe_symm :
       (TotalSpace.toProd H E).symm :=
   rfl
 
-theorem contMDiff_tangentBundleModelSpaceHomeomorph {n : ℕ∞} :
+theorem contMDiff_tangentBundleModelSpaceHomeomorph :
```

Comment:
If you split these into a separate PR, feel free to maintainer merge on my behalf already.